query-service-impl/src/test/java/org/hypertrace/core/query/service/QueryServiceImplTest.java

```diff
@@ -65,7 +65,6 @@ public void testBlankTenantColumnNameThrowsException() {
 
     // whitespace tenant id column name
     QueryServiceImplConfig queryServiceConfig2 = new QueryServiceImplConfig();
-    queryServiceConfig2.setTenantColumnName("   ");
```

Comment:
These tests are not valid anymore, are they? Also where are the tests to check for the empty tenant id now.

Reply:
you're right. Removed. Thanks for pointing this.

Reply:
`org.hypertrace.core.query.service.pinot.PinotBasedRequestHandlerTest#testNullTenantIdQueryRequestContextThrowsNPE`